tools/docker-network/README.md

```diff
@@ -0,0 +1,36 @@
+# GoShimmer Network with Docker
+
+![Docker network](../../images/docker-network.png)
+
+Running `docker-compose` spins up a GoShimmer network within Docker as schematically shown in the figure above.
+`N` defines the number of `peer_replicas` and can be specified when running the network.
+The peers can communicate freely within the Docker network 
+while the autopeering network visualizer, `master_peer's` dashboard and web API are reachable from the host system on the respective ports.
+
+The different containers (`entry_node`, `peer_master`, `peer_replica`) are based on the same config file 
+and separate config file and modified as necessary, respectively. 
+
+## How to use as development tool
+Using a standalone throwaway Docker network can be really helpful as a development tool. 
+
+Prerequisites: 
+- Docker
+- Docker compose
```

Comment:
As we already have such a nice README here, I think we should add that we are requiring >= 3.5